grpc/authn-private.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package grpc;
+option go_package = "grpc";
+
+import "google/protobuf/empty.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+
+message GetAccountRequest { string id = 1; }
+
+message GetAccountResponse {
+  int64 id = 1;
+  string username = 2;
+  bool locked = 3;
+  bool deleted = 4;
+}
+
+message GetAccountResponseEnvelope { GetAccountResponse result = 1; }
+
+message UpdateAccountRequest {
+  string id = 1;
+  string username = 2;
+}
+
+
+message LockAccountRequest { string id = 1; }
+
+message UnlockAccountRequest { string id = 1; }
+
+message ArchiveAccountRequest { string id = 1; }
+
+message ImportAccountRequst {
+  string username = 1;
+  string password = 2;
+  bool locked = 3;
+}
+message ImportAccountResponse { int64 id = 1 [ (gogoproto.jsontag) = "id" ]; }
+
+message ImportAccountResponseEnvelope { ImportAccountResponse result = 1; }
+
+
+message ExpirePasswordRequest { string id = 1; }
+
+message ServiceStatsResponse {
+    message active_stats {
+        map<string, int64> daily = 1;
+        map<string, int64> weekly = 2;
+        map<string, int64> monthly = 3;
+    }
+    active_stats actives = 1;
+}
+
+service AdminAuthN {
+
+  rpc GetAccount(GetAccountRequest) returns (GetAccountResponseEnvelope) {
+    option (google.api.http) = {
+      get : "/accounts/{id}",
+    };
+  }
+
+  rpc UpdateAccount(UpdateAccountRequest) returns (google.protobuf.Empty) {
```

Comment:
Sorry this is totally unsolicited feedback, but may I suggest that you use dedicated response types for each RPC, including those that return nothing? The types of your RPC request/response become part of the API, and later changing them (say you decide to add a value in the response) would mean breaking the API in a non-backward compatible manner.

So in general, it's preferable to have all RPC look like:

```protobuf
service Blah {
    rpc DoThing(DoThingRequest) returns (DoThingResponse);
}

message DoThingRequest {
    // stuff goes here
}

message DoThingsResponse { } // empty message
```
Also note that each RPC should have it's own `Request`  and  `Response` types, and not share them between RPCs (for the same reasons).